### ISSUE TITLE
Formatting toolbar controls missing when using a keyboard

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6910]
When using the mouse and clicking, for example, on a paragraph, the formatting toolbar appears with all its controls displayed:

![screen shot 2018-05-23 at 11 49 29](https://user-images.githubusercontent.com/1682452/40417261-a295436c-5e7f-11e8-9050-bf4e497ecddd.png)

Instead, when using the keyboard to navigate with the Tab key and the toolbar is focused, only a few controls are displayed. In the screenshots below, the focused element is highlighted with a red outline.

Tabbing into the toolbar: only 4 controls are initially displayed:

![screen shot 2018-05-23 at 11 40 41](https://user-images.githubusercontent.com/1682452/40417359-daa405d6-5e7f-11e8-9edd-ef293d1a45fa.png)

Press tab again to move to the last of these 4 controls:

![screen shot 2018-05-23 at 11 40 47](https://user-images.githubusercontent.com/1682452/40417422-01fed494-5e80-11e8-8d7b-0b876e623275.png)

Now press tab again: the missing controls appear but they're "skipped", as focus has already landed on the block content:

![screen shot 2018-05-23 at 11 40 52](https://user-images.githubusercontent.com/1682452/40417472-1e45f6dc-5e80-11e8-82ef-dab90a6a4afc.png)

The expected behavior would be that _all_ the toolbar controls are displayed when the toolbar receives focus.

Not sure what's happening, I can reproduce consistently on current master with different browsers. No JS errors.

**Comment by afercia** [comment]
Uploading an animated GIF to clarify, still happens on 3.0. Please notice in a first moment the toolbar has just a few buttons. Then, when tabbing through the toolbar, bold, italic, strikethrough, and link appear:

![gutenberg_6910](https://user-images.githubusercontent.com/1682452/40987223-581b0ca4-68e8-11e8-9d40-12a9aceb17b1.gif)

**Comment by afercia** [comment]
With regards to [the related conversation on Slack](https://wordpress.slack.com/archives/C02QB2JS7/p1528292974000429),  worth noting in the current Classic Editor there's no need to make a selection to have the bold, italic, strikethrough, and link buttons work. In fact, it is possible to just put the caret within a word, press (for example) bold and the whole word becomes bold.

Regardless, the caret must be within the editable area but I'm not sure not showing the buttons if the caret is _not_ within the editable area is the best user experience. Seems to me it's just a bit confusing.

**Comment by IreneStr** [comment]
We've discussed this on WCEU with @jorgefilipecosta and @enejb. It might be a good idea to pass a boolean as a prop to each block that determines whether the first editable should be automatically focussed when the block gains focus, for example when tabbing to it. That way, when a user tabs to a paragraph (or any other block for that matter), the rich text formatting buttons will always show up. If there are reasons to not focus the first editable, this can also be managed with this prop.

**Comment by joedolson** [comment]
Tested 10/16/2018 with NVDA/Firefox and this appears to work as expected now.

**Comment by designsimply** [comment]
Closing based on the latest reply. Thank you for testing @joedolson!
